app/code/Magento/Paypal/Block/Iframe.php

```diff
@@ -198,7 +198,7 @@ protected function _beforeToHtml()
     protected function _toHtml()
     {
         if ($this->_isAfterPaymentSave()) {
-            $this->setTemplate('hss/js.phtml');
+            $this->setTemplate('Magento_Paypal::hss/js.phtml');
```

Comment:
Please, revert fix in places when the template specified dynamically

Reply:
Done.